app/constants/forms.js

```diff
@@ -7,7 +7,11 @@ export const FieldTypes = {
     ESSAY: 5,
     BOOLEAN: 6,
     SECTIONHEADER: 7,
-    MULTI: 8
+    MULTI: 8,
```

Comment:
iirc, these actually shouldn't be needed since forms aren't built based on this and the types are loaded via the api? They should be defined [here](https://github.com/mhacks/mhacks-web/blob/master/config/default.js#L63)

Reply:
The configuration state isn't always defined by the time the function gets ran and I saw those constants existed on the front-end so I just tried to use those; I'm not exactly sure of a better way when the configuration isn't there off the start